site/docs/main/configuration.md

```diff
@@ -186,6 +187,31 @@ If present, the value of the `CONTOUR_NAMESPACE` environment variable is used as
 
 The `CONTOUR_NAMESPACE` environment variable is set via the [Downward API][6] in the Contour [example manifests][7].
 
+## Bootstrap Config File
+
+The boostrap configuration file is generated by an initContainer in the Envoy daemonset which configures the Envoy
+container to connect to Contour and receive configuration via xDS.
+
+### Arguments
+
+There are arguments that can be passed to `contour boostrap` that help configure how Envoy
+connects to Contour:
+
+| Arg | Default  | Description |
+|------------|----------|-------------|
+| <nobr>resources-dir</nobr> | "" | Defines the directory where resource files will be written.  |
```

Comment:
Using the `--resources-dir` format might make it clearer that these are standard command-line flags.